tests/integration/capabilities.bats

```diff
@@ -53,3 +53,32 @@ function teardown() {
 	[[ "${output}" == *"CapPrm:	0000000000200000"* ]]
 	[[ "${output}" == *"NoNewPrivs:	1"* ]]
 }
+
+@test "runc exec --cap" {
+	update_config '.process.terminal = false'
+	update_config '.process.args = ["sleep", "infinity"]'
+	update_config '.process.user = {"uid":0}'
+	update_config '.process.capabilities.bounding = ["CAP_KILL", "CAP_CHOWN", "CAP_SYSLOG"]'
+	update_config '.process.capabilities.effective = ["CAP_KILL"]'
+	update_config '.process.capabilities.permitted = ["CAP_KILL", "CAP_CHOWN"]'
+	update_config '.process.capabilities.inheritable = ["CAP_CHOWN", "CAP_SYSLOG"]'
+	update_config '.process.capabilities.ambient = ["CAP_CHOWN"]'
+	__runc run -d test_some_caps
+	[ "$status" -eq 0 ]
```

Comment:
You are not checking runc exit code here, because `__runc` doesn't use bats' `runc` which sets `$status`.

If you copied this code from some other place -- that place is also wrong.